### DISCUSSION TITLE
Content Editor Components - Build Icons

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#2897] [Request: Icon] [answered]
Hellooo me again. I thought I'd combine the content editor build icons into one ticket. Please let me know if it makes sense to discuss these in an icon-themed office hours, or async. The code icon is one that @brannytempy proposed for Segment, but i wasn't sure if it's been officially added to the library yet. Thanks!

![Screenshot 2022-12-15 at 3 28 44 PM](https://user-images.githubusercontent.com/87141551/207980716-f7b7595a-94ed-4929-a684-1b490e71cc1c.png)

**Comment by kelseycavitt** [comment]
@ceyerkes, awesome, Is this screenshot you’ve shared here final? If they are the icons you’d like to use, let's do an async round of feedback in Figma. [We did this with @brannytempy ](https://github.com/twilio-labs/paste/discussions/2840) when he was getting Segment icons added. We can then discuss if it makes sense to get those icons added into Paste. Please follow the [contribution guidelines](https://paste.twilio.design/introduction/contributing/icons).

For the “code” icon, Brandon proposed a simpler version. He’s also got an [Airtable](https://airtable.com/appZrG8GeTsIgJ3UI/tblEmoApPo4hpNJ20/viwLFvsnM1cN4zAyo?blocks=hide) of other icons he’s proposing to be added to get in parity with Segment icon needs. This might be helpful for you to look at! We gave him some early feedback on what could be used, so these have had some vetting already.

![CleanShot 2023-01-05 at 11 20 54](https://user-images.githubusercontent.com/1484014/210854039-a4515e2a-c208-40fe-bf46-b71de378c6a8.png)

Let us know when the Figma is ready for a review! And for prosperity sake, could you share the Figma link here as well? Thanks!

**Reply by ceyerkes** [reply]
Hey @kelseycavitt! Nope, these aren't final - was just looking for input/ideas. I'll set up an [async Figma situation](https://www.figma.com/file/AThoZNMoCXViDddWDbKb0d/Content-Editor?node-id=212%3A125804) and we can continue the discussion in there. I've poked around in Brandon's icon Airtable to make sure we're not duplicating efforts, and the only one I saw so far was the code one. Thank you!

**Reply by kelseycavitt** [reply]
Great!!

**Comment by ceyerkes** [comment]
Hello! I'm looking for async review of some [content editor build icons](https://www.figma.com/file/AThoZNMoCXViDddWDbKb0d/Content-Editor?node-id=212%3A125804&t=cuPt1oTGX0ZMc91k-1) i'm working on.

1. Alignment (left, right, center, full)
2. Image + text
3. Button
4. Divider
5. Spacer
6. Social
7. Unsubscribe (I'm cool with @kelseycavitt's suggestion, but I'm wondering if the line weight matches other icons?)

I'll have a few image editor icons (crop, rotate, flip horizontally/vert, etc.) to submit for review as well - please let me know if I should start a separate discussion, or post here. Thanks!

**Reply by libertycarlin** [reply]
Hey! I'd love to give you feedback, but having a hard time understanding which ones are the icons you're proposing? @ceyerkes

**Reply by libertycarlin** [reply]
left a couple comments in the figma!

**Reply by ceyerkes** [reply]
@libertycarlin sorry about that - I'll get in there and make it more clear, thanks!

**Comment by nkrantz** [comment, marked as answer]
Hey @ceyerkes, feel free to follow along with this PR (https://github.com/twilio-labs/paste/pull/3095). When it merges, all of the new icons will be available for use!
